quote.h

```diff
@@ -1,8 +1,10 @@
 #ifndef QUOTE_H
 #define QUOTE_H
 
-extern int quote_need();
-extern int quote();
-extern int quote2();
+#include "stralloc.h"
```

Comment:
As a good side effect, we notice more dependencies between source doe files.  Maybe will help to generate the Makefile later...